src/main/scala/workflow/graph/LabelEstimator.scala

```diff
@@ -0,0 +1,108 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+/**
+ * A LabelEstimator has a `fitRDDs` method which takes input data and input labels,
```

Comment:
The `fitRDDs` in LabelEstimator that needs to be implemented is typesafe:
`protected def fitRDDs(data: RDD[A], labels: RDD[L]): Transformer[A, B]`

What isn't type-safe is the already implemented overridden method from the EstimatorOperator:
```
  final override private[graph] def fitRDDs(inputs: Seq[DatasetExpression]): TransformerOperator = {
    fitRDDs(inputs(0).get.asInstanceOf[RDD[A]], inputs(1).get.asInstanceOf[RDD[L]])
  }
```